core/src/test/java/io/confluent/kafka/schemaregistry/SchemaRegistryTestHarness.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2025 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafka.schemaregistry;
+
+import java.util.Properties;
+
+/**
+ * Interface defining common operations needed by schema registry integration tests.
+ * This allows tests to be written against either Kafka-based or RP-based implementations.
```

Comment:
@akhileshm1 , @GunalKupta 
This is a public repository, so we should not refer to Confluent-internal terms or technologies